client-side/src/stories/tabs/tabs.css

```diff
@@ -0,0 +1,20 @@
+@import url("https://fonts.googleapis.com/css2?family=Nanum+Gothic&display=swap");
+.div .every-tabs,
+.div .profile-tabs,
```

Comment:
why css file? you should use scss files,
also please variables instead of colors hard-code

Reply:
את הקבצים האלו אמרת שאת לא מעלה,
את רוצה שאני בכל אופן ימחק אותם?

Reply:
I said that they should be in gitignore in order to ignore them